src/components/MessagesList/MessagesGroup/Message/MessagePart/Poll.vue

```diff
@@ -496,6 +486,10 @@ export default {
 	&__modal {
 		position: relative;
 		padding: 20px 20px 0 20px;
```

Comment:
```suggestion
		padding: 20px;
```

I'd make it this way, and remove bottom padding/margin from `poll__modal-actions` and `results__options`